### PR TITLE
Fix variable declarations/refs in JS Set object doc

### LINKED ISSUE #2263: Issue with "Set": Demo code throws unexpected errors due to variable declarations
MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

#### What information was incorrect, unhelpful, or incomplete?

The demonstration code threw uncaught errors. Variables were undeclared or referenced inconsistently.

#### Specific section or headline?

/*demo code quoted below throws several errors, mySet2  and set1, set2 are not declared */

mySet2 = new Set([1, 2, 3, 4])  
mySet2.size                    // 4
[...mySet2]                    // [1, 2, 3, 4]

// intersect can be simulated via
let intersection = new Set([...set1].filter(x => set2.has(x)))

// difference can be simulated via
let difference = new Set([...set1].filter(x => !set2.has(x)))

#### What did you expect to see?

//correction 1

let mySet2 = new Set([1, 2, 3, 4])  
console.log(mySet2.size)                    // 4
console.log([...mySet2])                    // [1, 2, 3, 4]

//correction 2

// intersect can be simulated via
let intersection = new Set([...mySet].filter(x => mySet2.has(x)))

// difference can be simulated via
let difference = new Set([...mySet].filter(x => !mySet2.has(x)))

#### Did you test this? If so, how?

ran demo code and corrected code on 
https://jscomplete.com/playground
produced correct console output after corrections were put in

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/set`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/javascript/reference/global_objects/set/index.html
* Last commit: https://github.com/mdn/content/commit/2ef7bb784bf1036ad7886b64e6451a68e573a44b
* Document last modified: 2021-01-15T08:25:52.000Z

</details>

### PR DESCRIPTION
**Pull request by CreaTorAlexander**
Fixes #2263 I wasn't sure about the console.log()'s and the naming. I named it now mySet1 and mySet2, I thought it's not good to name it mySet and mySet2

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/set/index.html [hunk 1]
```diff
@@ -81,32 +81,32 @@ <h2 id="Examples">Examples</h2>
 
 <h3 id="Using_the_Set_object">Using the Set object</h3>
 
-<pre class="brush: js notranslate">let mySet = new Set()
+<pre class="brush: js notranslate">let mySet1 = new Set()
```
Comment (avivmu) [inline files/en-us/web/javascript/reference/global_objects/set/index.html:84]: Also can use `const` since reference doesn't change, same with `mySet2`

Comment (CreaTorAlexander) [reply files/en-us/web/javascript/reference/global_objects/set/index.html:84]: Agree. But then we should also update setA, setB, setC, intersection, difference and the array to const.
How can I update now an existing pull request?

**Comment by avivmu** [comment]
Push to the same branch

File: files/en-us/web/javascript/reference/global_objects/set/index.html (hunk 1 above)
Comment (Ryuno-Ki) [reply files/en-us/web/javascript/reference/global_objects/set/index.html]: >  How can I update now an existing pull request?

Commit on the same branch and push your changes. GitHub will deal with the rest.

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**